front/public/locales/fr/infraEditor.json

```diff
@@ -81,8 +81,8 @@
     "description": "Une caténaire est un ensemble de câbles permettant d’alimenter les trains en utilisant un pantographe. La caténaire est identifiée par un id",
```

Comment:
«La caténaire est identifiée par un identifiant »

tautological

Reply:
I'll fix that !